share/locale/en.json

```diff
@@ -1,165 +1,165 @@
 {
-    "config_onion_service": "Configuring onion service on port {0:d}.",
-    "preparing_files": "Preparing files to share.",
+    "config_onion_service": "Setting up onion service on port {0:d}.",
+    "preparing_files": "Preparing OnionShare.",
     "wait_for_hs": "Waiting for HS to be ready:",
     "wait_for_hs_trying": "Trying…",
     "wait_for_hs_nope": "Not ready yet.",
-    "wait_for_hs_yup": "Ready!",
-    "give_this_url": "Give this address to the person you're sending the file to:",
-    "give_this_url_stealth": "Give this address and HidServAuth line to the person you're sending the file to:",
-    "give_this_url_receive": "Give this address to the people sending you files:",
-    "give_this_url_receive_stealth": "Give this address and HidServAuth line to the people sending you files:",
+    "wait_for_hs_yup": "Ready.",
+    "give_this_url": "Give this address to the recipient of the file:",
+    "give_this_url_stealth": "Give this address and HidServAuth line to the recipient of the file:",
+    "give_this_url_receive": "Give this address to those that send you files:",
+    "give_this_url_receive_stealth": "Give this address and your HidServAuth line to those that send you files:",
     "ctrlc_to_stop":  "Press Ctrl+C to stop the server",
     "not_a_file": "{0:s} is not a valid file.",
     "not_a_readable_file": "{0:s} is not a readable file.",
-    "no_filenames": "You must specify a list of files to share.",
-    "no_available_port": "Could not start the Onion service as there was no available port.",
-    "download_page_loaded": "Download page loaded",
-    "other_page_loaded": "Address loaded",
-    "close_on_timeout": "Stopped because timer expired",
-    "closing_automatically": "Stopped because download finished",
-    "timeout_download_still_running": "Waiting for download to complete",
+    "no_filenames": "Add files to OnionShare.",
```

Comment:
OnionShare is both a command line and GUI app, and this is actually a command line-only string. When you just run `onionshare` without specify any files, it displays:

```
$ ./dev_scripts/onionshare
OnionShare 2.0.dev | https://onionshare.org/
You must specify a list of files to share.
``` 

Although, now that I look at it, I think it's better to just remove this whole string, and instead display the full usage if you don't specify a list of files, like this:

```
$ ./dev_scripts/onionshare
OnionShare 2.0.dev | https://onionshare.org/
usage: onionshare [-h] [--local-only] [--stay-open] [--shutdown-timeout <int>]
                  [--stealth] [--receive] [--config config] [--debug]
                  [filename [filename ...]]

positional arguments:
  filename                  List of files or folders to share

optional arguments:
  -h, --help                show this help message and exit
  --local-only              Do not attempt to use Tor: For development only
  --stay-open               Keep onion service running after download has
                            finished
  --shutdown-timeout <int>  Shut down the onion service after N seconds
  --stealth                 Create stealth onion service (advanced)
  --receive                 Receive files instead of sending them
  --config config           Path to a custom JSON config file (optional)
  --debug                   Log application errors to stdout, and log web
                            errors to disk
```

I just removed this string in #777.

Reply:
On closed introspection, I think what I meant to do was "Add files to OnionShare them".
Your version is much improved.